### PR TITLE
OOP home work

### PR DESCRIPTION
**Pull request by DankevichMisha**

### CODE REVIEW

**Comment by DankevichMisha** [comment]
ок

File: jv-oop12/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# jv-oop
```
Comment (bhdnchui) [inline jv-oop12/README.md:1]: Redundant file.

File: jv-oop12/checklist.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## Common mistakes (jv-oop)
```
Comment (bhdnchui) [inline jv-oop12/checklist.md:1]: Redundant file.

File: jv-oop12/src/main/java/core/basesyntax/Bulldozer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @java.lang.Override
```
Comment (bhdnchui) [inline jv-oop12/src/main/java/core/basesyntax/Bulldozer.java:4]:
```suggestion
    @Override
```

File: jv-oop12/src/main/java/core/basesyntax/MainApp.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
```
Comment (bhdnchui) [inline jv-oop12/src/main/java/core/basesyntax/MainApp.java:5]: Make constant.

File: jv-oop12/src/main/java/core/basesyntax/String.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class String {
+}
```
Comment (bhdnchui) [inline jv-oop12/src/main/java/core/basesyntax/String.java:4]:
```suggestion

```

File: jv-oop12/src/test/java/core/basesyntax/MainAppTest.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public class MainAppTest {
+
+}
```
Comment (bhdnchui) [inline jv-oop12/src/test/java/core/basesyntax/MainAppTest.java:5]:
```suggestion

```

**Review by bhdnchui — CHANGES_REQUESTED** [review]

File: jv-oop12/README.md (hunk 1 above)
Comment (DankevichMisha) [reply jv-oop12/README.md:1]: okay

File: jv-oop12/checklist.md (hunk 2 above)
Comment (DankevichMisha) [reply jv-oop12/checklist.md:1]: ok

File: jv-oop12/src/main/java/core/basesyntax/MainApp.java (hunk 4 above)
Comment (DankevichMisha) [reply jv-oop12/src/main/java/core/basesyntax/MainApp.java:5]: like a :
final Machine[] machines = new Machine[3]; ?

File: jv-oop12/src/main/java/core/basesyntax/String.java (hunk 5 above)
Comment (DankevichMisha) [reply jv-oop12/src/main/java/core/basesyntax/String.java:4]: delete this?

**Comment by DankevichMisha** [comment]
+

**Comment by DankevichMisha** [comment]
correct mistakes

**Review by Rommelua — CHANGES_REQUESTED** [review]
Delete this folder jv-oop12 - we do not need it
image

**Review by DankevichMisha — COMMENTED** [review]
//

**Review by DankevichMisha — COMMENTED** [review]
вроде бы оно

**Review by Rommelua — CHANGES_REQUESTED** [review]
Треба видалити зайві папки

**Pull request closed without merging**